docs/source/API/containers/ScatterView.rst

```diff
@@ -6,14 +6,77 @@
+
+
 .. _parallelReduce: ../core/parallel-dispatch/parallel_reduce.html
 
 .. |parallelReduce| replace:: :cpp:func:`parallel_reduce`
 
+.. _View: ../core/view/view.html
+
+.. |View| replace:: ``View``
+
+.. |reset| replace:: ``reset()``
+
+.. |access| replace:: ``access()``
+
+.. |contribute| replace:: ``contribute()``
+
+.. |create_scatter_view| replace:: ``create_scatter_view()``
+
+Usage
+-----
+
+A Kokkos ScatterView serves as an interface for a standard Kokkos::|View|_ and allow access to it either via Atomic or Data Replication based scatter algorithms, choosing the strategy that should be the fastest for the ScatterView Execution Space.
+
+The best option to create a ScatterView is to make a call to the free function Kokkos::Experimental::|create_scatter_view|_.
+
+Construction of a ScatterView can be expensive, so you should try to reuse the same one if possible, in which case, you should call |reset|_ between uses.
+
+ScatterView can not be addressed directly: each thread inside a parallel region needs to make a call to |access|_ and access the underlying View through the return value of |access|_.
+
+Following the parallel region, a call to the free function Kokkos::Experimental::|contribute|_ should be made to perform the final reduction.
+
+Interface
+---------
+.. code-block:: cpp
+
+    template <typename DataType [, typename Layout [, typename ExecSpace [, typename Operation [, typename Duplication [, typename Contribution]]]]]>
+    class ScatterView
+
+Parameters
+~~~~~~~~~~
+Template parameters other than ``DataType`` are optional, but if one is specified, preceding ones must also be specified.
+That means for example that ``Operation`` can be omitted but if it is specified, ``Layout`` and ``ExecSpace`` must also be specified.
+
+* ``DataType``, ``Layout`` and ``ExecSpace`` need to be the same types as the one from the Kokkos::View this ScatterView is interfacing.
+
+* ``Operation``:
+  Can take the values:
+
+  - ``Kokkos::Experimental::ScatterSum``: performs a Sum. It is the default value.
+
+  - ``Kokkos::Experimental::ScatterProd``: performs a Multiplication.
+
+  - ``Kokkos::Experimental::ScatterMin``: takes the min.
+
+  - ``Kokkos::Experimental::ScatterMax``: takes the max.
+
+* ``Duplication``:
+  Whether to duplicate the grid or not; defaults to ``Kokkos::Experimental::ScatterDuplicated``, other option is ``Kokkos::Experimental::ScatterNonDuplicated``.
+
+* ``Contribution``:
+  Whether to contribute to use atomics; defaults to ``Kokkos::Experimental::ScatterAtomics``, other option is ``Kokkoss::Experimental::ScatterNonAtomic``.
+
+Creating a ScatterView with non default ``Operation``, ``Duplication`` or ``Contribution`` using this interface can become complicated, because you need to explicit the exact type for ``DataType``, ``Layout`` and ``ExecSpace``. This is why it is advised that you instead use the function Kokkos::Experimental::|create_scatter_view|_.
```

Comment:
```suggestion
Creating a ScatterView with non default ``Operation``, ``Duplication`` or ``Contribution`` using this interface can become complicated, because you need to specify the exact type for ``DataType``, ``Layout`` and ``ExecSpace``. This is why it is advised that you instead use the function Kokkos::Experimental::|create_scatter_view|_.
```